ads/google/a4a/test/test-google-ads-a4a-config.js

```diff
@@ -310,6 +312,261 @@ describe('a4a_config', () => {
       }
     });
   });
+  it('should serve Delayed Fetch to unlaunched DoubleClick filler',
+    () => {
+      toggleExperiment(win, 'a4aFastFetchDoubleclickLaunched', false, true);
+      element.setAttribute('type', 'doubleclick');
+      let externalBranches, internalBranches;
+      if (isExperimentOn(win, 'a4aFastFetchDoubleclickLaunched')) {
+        externalBranches =
+          expts.DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES_POST_LAUNCH;
+        internalBranches =
+          expts.DOUBLECLICK_A4A_INTERNAL_EXPERIMENT_BRANCHES_POST_LAUNCH;
+      } else {
+        externalBranches =
+          expts.DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES_PRE_LAUNCH;
+        internalBranches =
+          expts.DOUBLECLICK_A4A_INTERNAL_EXPERIMENT_BRANCHES_PRE_LAUNCH;
+      }
+      expect(googleAdsIsA4AEnabled(win, element, 'expDoubleclickA4A',
+        externalBranches, internalBranches)).to.be.false;
+    });
+  it('should serve Delayed Fetch to unlaunched DoubleClick control',
+    () => {
+      toggleExperiment(win, 'a4aFastFetchDoubleclickLaunched', false, true);
+      toggleExperiment(win, 'expDoubleclickA4A', true, true);
+      element.setAttribute('type', 'doubleclick');
+      win.pageExperimentBranches['expDoubleclickA4A'] =
```

Comment:
This feels like tinkering with an implementation detail.  Do you need to modify the page experiment state directly?  Shouldn't `googleAdsIsA4AEnabled` do that for you, via `randomlySelectUnsetPageExperiments`?  Or are you trying to simulate the effect of having already run `maybeSetExperimentFromUrl`?  If the latter, I recommend using the getters/setters in `traffic-experiments.js` to manipulate this state instead.